From 384ba5040f54fe87fe6e8b5b2c453a7033ef60a1 Mon Sep 17 00:00:00 2001
From: Christian Marangi <ansuelsmth@gmail.com>
Date: Fri, 22 Aug 2025 15:43:56 +0200
Subject: [PATCH] an7581: enable DM_MDIO and MDIO CMD by default

Enable DM_MDIO and MDIO CMD in the default config.

Signed-off-by: Christian Marangi <ansuelsmth@gmail.com>
---
 configs/an7581_evb_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/configs/an7581_evb_defconfig b/configs/an7581_evb_defconfig
index 5765dcbaaec..a9b4387179a 100644
--- a/configs/an7581_evb_defconfig
+++ b/configs/an7581_evb_defconfig
@@ -81,6 +81,9 @@ CONFIG_SPI=y
 CONFIG_DM_SPI=y
 CONFIG_SHA512=y
 CONFIG_AIROHA_ETH=y
+CONFIG_DM_MDIO=y
+CONFIG_CMD_MII=y
+CONFIG_CMD_MDIO=y
 CONFIG_MMC_MTK=y
 CONFIG_AIROHA_SNFI_SPI=y
 CONFIG_CMD_UBI=y
-- 
2.50.0

